packages/Telephony/Actions/TimeoutChoiceInput.cs

```diff
@@ -104,6 +107,11 @@ public override async Task<DialogTurnResult> ContinueDialogAsync(DialogContext d
                 //End dirty hack
             }
 
+            return await ContinueTimeoutChoiceInputDialogAsync(dc, cancellationToken).ConfigureAwait(false);
+        }
+
+        protected virtual async Task<DialogTurnResult> ContinueTimeoutChoiceInputDialogAsync(DialogContext dc, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
I believe the code here can be made terser by removing `CancellationToken)`

```suggestion
        protected virtual async Task<DialogTurnResult> ContinueTimeoutChoiceInputDialogAsync(DialogContext dc, CancellationToken cancellationToken = default)
```